code/frameworks/sveltekit/template/stories_svelte-kit-skeleton-ts/Environment.svelte

```diff
@@ -1,8 +1,7 @@
 <script>
-	import { browser, dev, building, version } from '$app/environment';
+  import { browser, dev, building, version } from '$app/environment';
 </script>
 
 <div data-testid="browser">{browser}</div>
 <div data-testid="dev">{dev}</div>
 <div data-testid="building">{building}</div>
-<div data-testid="version">{version}</div>
```

Comment:
I wonder if we could leave the import (and maybe add a console log just to avoid finding ourselves in the situation where version is no longer importable and we don't notice? This should solve the chromatic issue without getting rid of the "value" of the test

Reply:
Good call. I'll add it back but ignore it in chromatic